its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -296,6 +281,16 @@ public static void deleteLocalCache() {
     }
   }
 
+  // Populate the plugin cache before the tests to avoid race conditions in the begin step
+  public static void warmupLocalCache(Orchestrator orch, TemporaryFolder temp) throws IOException {
```

Comment:
Or `populate` or `init`
```suggestion
  public static void warmUpLocalCache(Orchestrator orch, TemporaryFolder temp) throws IOException {
```